releasenotes/notes/optimize-gatewayPortNotOnWorkload.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+releaseNotes:
+  - |
+    **Optimized** Replace `IST0104 - GatewayPortNotOnWorkload` to new message `IST0162 - GatewayPortNotDefinedOnService` alert in 
```

Comment:
nit: I think the release note can just be something like:
```
**Added** IST0162 `GatewayPortNotDefinedOnService` message to detect the issue where Gateway port was not exposed by Service.
```

Reply:
great idea